packages/components/inputs/select-utils/src/create-select-styles.ts

```diff
@@ -115,11 +146,6 @@ const controlStyles = (props: TProps) => (base: TBase, state: TState) => {
       : {}),
 
     '&:focus-within': {
-      boxShadow: (() => {
```

Comment:
Removing this since it creates unnecessary `3px` green border when focused and in error or warning states. We just need a `2px` border here.

https://github.com/commercetools/ui-kit/assets/52276952/1217d217-7153-4361-8018-1f823b35148d